api-spec/protobuf/operator.proto

```diff
@@ -362,3 +369,34 @@ message WebhookInfo {
   string endpoint = 2;
   bool is_secured = 3;
 }
+
+
+message ListDepositsRequest{
+  int64 account_index = 1;
+  Page page = 2;
+}
+message ListDepositsReply{
+  int64 account_index = 1;
+  repeated TxOutpoint deposits = 2;
+}
+
+message ListWithdrawalsRequest{
+  int64 account_index = 1;
+  Page page = 2;
+}
+message ListWithdrawalsReply{
+  int64 account_index = 1;
+  repeated Withdrawal withdrawals = 2;
+}
+
+message Withdrawal {
```

Comment:
this message should include also the txid of the withdrawal tx

Reply:
Done